lib/sles4sap/ipaddr2.pm

```diff
@@ -1337,7 +1368,7 @@ sub ipaddr2_cluster_create {
         cmd => 'sudo crm --version',
         bastion_ip => $args{bastion_ip});
     ipaddr2_ssh_internal(id => 1,
-        cmd => 'zypper se -s -i crmsh',
+        cmd => 'sudo zypper se -s -i crmsh',
```

Comment:
Do we need sudo?

Reply:
yes, same reason as `sudo zypper se -s -i cloud-init`